### PR TITLE
Add button component(s)

### LINKED ISSUE #19: Create components: buttons

### PR DESCRIPTION
**Pull request by suneettipirneni**
Adds the button components as seen in the figma components page. Closes #19

### CODE REVIEW

File: client/src/scss/button.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    color: white;
+    transition: 0.5s;
+  }
+}
+
+.btn {
+  padding: 10px;
+  border-radius: 10px;
+  font-weight: 600;
+  font-size: 18px;
```
Comment (ctcuff) [inline client/src/scss/button.scss:19]: Super opinionated but I think 16px would fit a bit better here, also feel like 10px is a bit large for a `border-radius`, maybe 6px instead?

File: client/src/components/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  variant: 'primary',
+  children: [],
+  onClick: () => {},
+};
+
+Button.propTypes = {
+  variant: PropTypes.string,
+  children: PropTypes.node,
+  onClick: PropTypes.func,
+};
```
Comment (ctcuff) [inline client/src/components/Button.jsx:22]: Might possibly want to add a `className` prop? Maybe it would come in handy in the future

**Review by ctcuff — APPROVED** [review]
A few really small suggestions but looks good! Also noticed some compiled CSS for `App` and `button` managed to sneak in :eyes:

File: client/src/components/Button.jsx (hunk 2 above)
Comment (suneettipirneni) [reply client/src/components/Button.jsx:30]: Good idea!

**Pull request merged**